### PR TITLE
Photography faker.

### PR DESCRIPTION
**Pull request by L-Evg**

### CODE REVIEW

File: src/main/java/com/github/javafaker/Photography.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+     */
+    public String lens() {
+        return faker.fakeValuesService().fetchString("photography.lens");
+    }
+
+    /**
+     * @return a photography genre.
+     */
+    public String gengre() {
+        return faker.fakeValuesService().fetchString("photography.gengre");
```
Comment (patou) [inline src/main/java/com/github/javafaker/Photography.java:46]:
```suggestion
    public String genre() {
        return faker.fakeValuesService().fetchString("photography.genre");
```

File: src/main/resources/en/photography.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      lens: ["500mm/8", "17mm/2.0", "35mm/1.8", "50mm/1.2", "50mm/1.4", "50mm/1.8", "17-85mm/4.0-8.0", "85mm/1.2", "40mm/2.8", "70–300mm/4.5–5.6", "300mm/2.8", "35-350mm/3.5-5.6", "75-300mm/4-5.6", "24mm/1.4", "17–40mm/4", "24–105mm/4", "70–200mm/2.8", "70–200mm/4", "200-400mm/4", "85mm/1.8", "135mm/4", "600mm/4", "800mm/5.6", "18–55mm/3.5–5.6", "55–250mm/4–5.6", "10-24mm/3.5-4.5", "12-24mm/4", "16-85mm/3.5-5.6", "17-55mm/2.8", "18-55mm/3.5-5.6", "18-70mm/3.5-4.5", "18-105mm/3.5-5.6", "18-140mm/3.5-5.6", "18-200mm/3.5-5.6", "18-300mm/3.5-5.6", "18-300mm/3.5-6.3", "55-200mm/4-5.6", "fisheye", "tele", "wide-angle"]
+      gengre: ["Abstract", "Portrait", "Landscape", "Techno", "Transportation", "Aviation", "Trains", "Ships", "Automobiles", "Bikes", "Army", "Navy", "Flora", "Fauna", "Sports", "Street", "City", "Village", "Country side", "Architecture", "Interior", "Food", "Wedding", "Stock", "Paparazzi", "Social", "Fashion", "Beauty", "Nude", "Science", "Space", "Aerial", "Industrial", "Artistic", "Underwater", "Night", "Low key", "High key", "Lomo", "Panoramic", "Macro", "Micro", "nano"]
```
Comment (patou) [inline src/main/resources/en/photography.yml:27]:
```suggestion
      genre: ["Abstract", "Portrait", "Landscape", "Techno", "Transportation", "Aviation", "Trains", "Ships", "Automobiles", "Bikes", "Army", "Navy", "Flora", "Fauna", "Sports", "Street", "City", "Village", "Country side", "Architecture", "Interior", "Food", "Wedding", "Stock", "Paparazzi", "Social", "Fashion", "Beauty", "Nude", "Science", "Space", "Aerial", "Industrial", "Artistic", "Underwater", "Night", "Low key", "High key", "Lomo", "Panoramic", "Macro", "Micro", "nano"]
```

File: src/main/resources/ru.yml [hunk 3]
```diff
@@ -74,3 +74,26 @@ ru:
...
+    lens: ["500мм/8", "17мм/2.0", "35мм/1.8", "50мм/1.2", "50мм/1.4", "50мм/1.8", "17-85мм/4.0-8.0", "85мм/1.2", "40мм/2.8", "70–300мм/4.5–5.6", "300мм/2.8", "35-350мм/3.5-5.6", "75-300мм/4-5.6", "24мм/1.4", "17–40мм/4", "24–105мм/4", "70–200мм/2.8", "70–200мм/4", "200-400мм/4", "85мм/1.8", "135мм/4", "600мм/4", "800мм/5.6", "18–55мм/3.5–5.6", "55–250мм/4–5.6", "10-24мм/3.5-4.5", "12-24мм/4", "16-85мм/3.5-5.6", "17-55мм/2.8", "18-55мм/3.5-5.6", "18-70мм/3.5-4.5", "18-105мм/3.5-5.6", "18-140мм/3.5-5.6", "18-200мм/3.5-5.6", "18-300мм/3.5-5.6", "18-300мм/3.5-6.3", "55-200мм/4-5.6", "рыбий глаз", "теле", "широкоугольный"]
+    gengre: ["Абстракция", "Портрет", "Пейзаж", "Техника", "Транспорт", "Авиация", "Поезда", "Корабли", "Автомобили", "Велосипеды", "Армейское", "ВМФ", "Флора", "Фауна", "Спорт", "Улица", "Город", "Деревня", "за городом", "Архитектура", "Интерьер", "Пища", "Свадьба", "Из готового", "Папарацци", "Общественное", "Мода", "Красота", "Ню", "Наука", "Космос", "Воздушное", "Промышленное", "Искусство", "Под водой", "Ночь", "чёрный ключ", "белый ключ", "ЛОМО", "Панорама", "Макро", "Микро", "Нано"]
```
Comment (patou) [inline src/main/resources/ru.yml:98]:
```suggestion
    genre: ["Абстракция", "Портрет", "Пейзаж", "Техника", "Транспорт", "Авиация", "Поезда", "Корабли", "Автомобили", "Велосипеды", "Армейское", "ВМФ", "Флора", "Фауна", "Спорт", "Улица", "Город", "Деревня", "за городом", "Архитектура", "Интерьер", "Пища", "Свадьба", "Из готового", "Папарацци", "Общественное", "Мода", "Красота", "Ню", "Наука", "Космос", "Воздушное", "Промышленное", "Искусство", "Под водой", "Ночь", "чёрный ключ", "белый ключ", "ЛОМО", "Панорама", "Макро", "Микро", "Нано"]
```

File: src/test/java/com/github/javafaker/PhotographyTest.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    @Test
+    public void lens() {
+        String value = faker.photography().lens();
+        assertNonNullOrEmpty(value);
+    }
+
+    @Test
+    public void gengre() {
+        String value = faker.photography().gengre();
+        assertNonNullOrEmpty(value);
```
Comment (patou) [inline src/test/java/com/github/javafaker/PhotographyTest.java:44]:
```suggestion
    public void genre() {
        String value = faker.photography().genre();
        assertNonNullOrEmpty(value);
```

**Review by patou — COMMENTED** [review]
I thinks there are a typo error with gengre ?

File: src/test/java/com/github/javafaker/PhotographyTest.java (hunk 4 above)
Comment (L-Evg) [reply src/test/java/com/github/javafaker/PhotographyTest.java:44]: @patou -  merci pour votre commentaire! And the suggestions.

**Pull request merged**